zerver/models.py

```diff
@@ -366,10 +366,16 @@ class Realm(models.Model):
     DEFAULT_TOPIC_EDITING_LIMIT_SECONDS = 259200
 
     EDIT_TOPIC_POLICY_EVERYONE = 1
+    EDIT_TOPIC_POLICY_MEMBERS_ONLY = 2
+    EDIT_TOPIC_POLICY_FULL_MEMBERS_ONLY = 3
+    EDIT_TOPIC_POLICY_MODERATORS_ONLY = 4
     EDIT_TOPIC_POLICY_ADMINS_ONLY = 5
 
     EDIT_TOPIC_POLICY_TYPES = [
         EDIT_TOPIC_POLICY_EVERYONE,
+        EDIT_TOPIC_POLICY_MEMBERS_ONLY,
+        EDIT_TOPIC_POLICY_FULL_MEMBERS_ONLY,
+        EDIT_TOPIC_POLICY_MODERATORS_ONLY,
         EDIT_TOPIC_POLICY_ADMINS_ONLY,
     ]
```

Comment:
Can we do an additional commit at the end to change these to the COMMON_POLICY_TYPES?